tests/virt_autotest/virt_utils.pm

```diff
@@ -219,6 +219,11 @@ sub handle_sp_in_settings_with_sp0 {
 sub update_guest_configurations_with_daily_build {
     repl_repo_in_sourcefile;
     repl_module_in_sourcefile;
+    #workaround of bsc#1177790
+    if (is_sle('=15-sp2')) {
+        script_run "sed -i 's/openqa.suse.de/10.160.0.207/g' /usr/share/qa/virtautolib/data/sources.*";
```

Comment:
Can you record_info for this workaround?

Reply:
sure, I am also thinking of changing to modify the settings in named configuration files.